stress/common/src/main/java/alluxio/stress/worker/WorkerBenchParameters.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress.worker;
+
+import alluxio.stress.Parameters;
+
+import com.beust.jcommander.Parameter;
+
+/**
+ * Parameters used in the UFS I/O throughput test.
+ * */
+public class WorkerBenchParameters extends Parameters {
+  @Parameter(names = {"--threads"}, description = "the number of threads to use")
+  public int mThreads = 16;
+
+  @Parameter(names = {"--io-size"},
+          description = "size of data to write or read in total, in MB")
+  public int mDataSize = 4096;
```

Comment:
consider making this a String and using `FormatUtils` to parse a byte instead. Then users can specify 1k, 4k, 10M, 10g, etc instead. I think it is more user friendly.

Reply:
This is a cool idea. However I think using an integer in the unit of MB makes the most sense for this test. It doesn't really mean much running an I/O with KB and i think using int gets us away from annoying decimal sizes. I lean towards keeping it this way. What do you think?

Reply:
I think Zac was saying to make this a string value, we can we specify like `4m`. In the code, we can convert it to an int with `FormatUtils.parseSpaceSize` like https://github.com/Alluxio/alluxio/blob/master/stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java#L239

Reply:
Updated using formatted String now instead of int.